app/src/test/java/nl/tudelft/b_b_w/controller/BlockControllerUnitTest.java

```diff
@@ -103,7 +104,7 @@ public void testAddDupBlocks() {
     @Test(expected=RuntimeException.class)
     public void alreadyRevoked() {
         Block newBlock = BlockFactory.getBlock(TYPE_REVOKE, owner, sequenceNumber, ownHash,
```

Comment:
You should not be able to give a sequencenumber to the constructor upon creating a block